e2e/Tests/provisioning/ProvisioningE2ETests.cs

```diff
@@ -678,7 +678,6 @@ public async Task DPS_Registration_AmqpWs_X509_IndividualEnrollment_InvalidGloba
                 auth,
                 clientOptions);
 
-            using var cts = new CancellationTokenSource(PassingTimeoutMilliseconds);
```

Comment:
This gave us 1 big timeout. If a single operation failed due to timeout, we would not try again. I broke it up so we'd get multiple attempts, with each attempt being a reasonable duration.